docs/topics/installing-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-extension_{context}"]
+= Installing the {ProductShortName} extension for IntelliJ IDEA
+
+You can install the {ProductShortName} extension in both the IntelliJ IDEA Ultimate and the IntelliJ IDEA Community Edition.
```

Comment:
This looks verbose. Suggestion: "in the Ultimate and the Community Edition releases of IntelliJ IDEA". releases or versions or whatever distinguishes them.

I think you also need to decide whether you want to say "in IntelliJ IDEA" or "in the IntelliJ IDEA". Since I have no idea what IDEA stands for, you'll have to decide.

Reply:
Perhaps IDEA should be spelled out in full the first time in a module.

Reply:
No need. IDEA is just a cute way of saying IDE. The A does not stand for anything.

Reply:
Update for Ultimate and the Community Edition. Change this to "Ultimate and Community editions of IntelliJ IDEA."

https://www.jetbrains.com/idea/download/#section=linux